charts/argocd-apps/templates/wbaas-ui.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: ui
+  namespace: argocd
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+spec:
+  destination:
+    namespace: default
+    server: {{ .Values.clusterUrl }}
+  project: {{ .Values.environment }}
+  sources:
+    - repoURL: {{ .Values.repoUrls.charts }}
+      path: charts/ui
+      targetRevision: HEAD
+      helm:
+        valueFiles:
+          - $values/k8s/argocd/{{ .Values.environment }}/ui.values.yaml
+    - repoURL: {{ .Values.repoUrls.deploy }}
+      targetRevision: HEAD
```

Comment:
if you want to test this PR, change this to `de/argo-ui-simple`

Reply:
It's very easy to get lost in this but actually changing this on your local machine won't help. It's necessary to update this in argo after pausing auto-updating (in case anyone else needs to test this)